### ISSUE TITLE
syslog import causes failures on Appveyor

### ISSUE THREAD

**Issue by pganssle** [spulec/freezegun#214]
Just recently, our Appveyor builds (which were working before) started failing due to `freeze_gun` trying to import `syslog`. I'm not sure if this is a bug purely on the Appveyor side or what - it's not really clear to me why `freezegun` is trying to import `syslog` on Windows at all.

Here's one failure on [a build that is failing](https://ci.appveyor.com/project/dateutil/dateutil/build/2.3.823/job/01qcd77i23scpyus):

```python
================================== FAILURES ===================================
_____________________________ UtilsTest.testToday _____________________________
c:\python27\lib\site-packages\freezegun\api.py:494: in wrapper
    with self:
c:\python27\lib\site-packages\freezegun\api.py:365: in __enter__
    return self.start()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <freezegun.api._freeze_time object at 0x03B419B0>
          [ ... trimmed ... ]
    
        with warnings.catch_warnings():
            warnings.filterwarnings('ignore')
    
            for mod_name, module in list(sys.modules.items()):
                if mod_name is None or module is None:
                    continue
                elif mod_name.startswith(self.ignore):
                    continue
                elif (not hasattr(module, "__name__") or module.__name__ in ('datetime', 'time')):
                    continue
>               for module_attribute in dir(module):
E               ImportError: No module named syslog
c:\python27\lib\site-packages\freezegun\api.py:421: ImportError
```

The failure is happening [here](https://github.com/spulec/freezegun/blob/503f6e49c6d0f8cd775f1b73d46139cde232c248/freezegun/api.py#L427). I'm a bit puzzled about this, though. Why would `sys.modules.items()` contain `syslog` if it doesn't exist on that platform, and why is the `ImportError` firing on `dir(module)`?

**Comment by Kwpolska** [comment]
This issue seems to be caused by `py` in 1.5.x versons, as rolling back to `1.4.34` works. I reported this as pytest-dev/py#169.

**Comment by Kwpolska** [comment]
This has been fixed in py 1.5.2, at least for me.

**Comment by pganssle** [comment]
Fixed upstream. Thanks @Kwpolska for referencing the py ticket.
